apps/financial-aid/web-osk/src/routes/status/AidAmountCalculations/aidAmountCalculation.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useContext } from 'react'
+import { Text } from '@island.is/island-ui/core'
+
+import {
+  Estimation,
+  ContentContainer,
+  Footer,
+  StatusLayout,
+} from '@island.is/financial-aid-web/osk/src/components'
+
+import * as styles from './aidAmountCalculation.treat'
+
+import { UserContext } from '@island.is/financial-aid-web/osk/src/components/UserProvider/UserProvider'
+
+const AidAmountCalculations = () => {
+  const { user } = useContext(UserContext)
+
+  return (
+    <StatusLayout>
+      <ContentContainer>
+        {user?.activeApplication && (
+          <Estimation
+            homeCircumstances={user.activeApplication[0].homeCircumstances}
+            usePersonalTaxCredit={
+              user.activeApplication[0].usePersonalTaxCredit
```

Comment:
We can skip checking the zero value of the array here once we change the contract